From: Antonio Radici <antonio@debian.org>
Date: Thu, 27 Feb 2014 16:31:48 +0100
Subject: smime.rc

Add CA's provided by ca-certificates to the ones
distributed by the default installation of Mutt.

Gbp-Pq: Topic misc
---
 contrib/smime.rc | 19 ++++++++++---------
 1 file changed, 10 insertions(+), 9 deletions(-)

diff --git a/contrib/smime.rc b/contrib/smime.rc
index 5376089..efb0b6e 100644
--- a/contrib/smime.rc
+++ b/contrib/smime.rc
@@ -4,33 +4,34 @@
 
 # If you compiled mutt with support for both PGP and S/MIME, PGP
 # will be the default method unless the following option is set
-set smime_is_default
+#set smime_is_default
 
 # Uncoment this if you don't want to set labels for certificates you add.
 # unset smime_ask_cert_label
 
 # Passphrase expiration
-set smime_timeout=300
+#set smime_timeout=300
 
 # Global crypto options -- these affect PGP operations as well.
-set crypt_autosign = yes
-set crypt_replyencrypt = yes
-set crypt_replysign = yes
-set crypt_replysignencrypted = yes
-set crypt_verify_sig = yes
+#set crypt_autosign = yes
+#set crypt_replyencrypt = yes
+#set crypt_replysign = yes
+#set crypt_replysignencrypted = yes
+#set crypt_verify_sig = yes
 
 # Section A: Key Management.
 
 # The (default) keyfile for signing/decrypting.  Uncomment the following
 # line and replace the keyid with your own.
-set smime_default_key="12345678.0"
+#set smime_default_key="12345678.0"
 
 # Uncommen to make mutt ask what key to use when trying to decrypt a message.
 # It will use the default key above (if that was set) else.
 # unset smime_decrypt_use_default_key
 
 # Path to a file or directory with trusted certificates
-set smime_ca_location="~/.smime/ca-bundle.crt"
+#set smime_ca_location="~/.smime/ca-bundle.crt"
+set smime_ca_location=`for f in $HOME/.smime/ca-certificates.crt $HOME/.smime/ca-bundle.crt /etc/ssl/certs/ca-certificates.crt ; do if [ -e $f ] ; then echo $f ; exit ; fi ; done`
 
 # Path to where all known certificates go. (must exist!)
 set smime_certificates="~/.smime/certificates"
